src/bucket.js

```diff
@@ -70,23 +69,26 @@ export default class Bucket {
    * @param  {Boolean} [options.safe]    The safe option.
    * @return {Collection}
    */
-  collection(name, options={}) {
-    return new Collection(this.client, this, name, this._bucketOptions(options));
+  collection(name, options = {}) {
+    return new Collection(
+      this.client,
+      this,
+      name,
+      this._bucketOptions(options)
+    );
```

Comment:
You mean, you're not crazy about `this` in JavaScript? Yeah, me neither.

Sorry. Yeah this is slightly odd, but *at least* it's consistent :)